wix/Build.OpenJDK8.withHotspot.cmd

```diff
@@ -0,0 +1,101 @@
+    REM "!WIX!bin\candle.exe" -arch !PLATFORM! Main.wxs Files-!OUTPUT_BASE_FILENAME!.wxs Src-!OUTPUT_BASE_FILENAME!.wxs -ext WixUIExtension -ext WixUtilExtension -dProductSku="!PRODUCT_SKU!" -dProductMajorVersion="!PRODUCT_MAJOR_VERSION!" -dProductMinorVersion="!PRODUCT_MINOR_VERSION!" -dProductMaintenanceVersion="!PRODUCT_MAINTENANCE_VERSION!" -dProductPatchVersion="!PRODUCT_PATCH_VERSION!" -dProductId="!PRODUCT_ID!" -dReproDir="!REPRO_DIR!" -dSetupResourcesDir="!SETUP_RESOURCES_DIR!" -dCulture="!CULTURE!"
+    REM "!WIX!bin\light.exe" Main.wixobj Files-!OUTPUT_BASE_FILENAME!.wixobj Src-!OUTPUT_BASE_FILENAME!.wixobj -ext WixUIExtension -ext WixUtilExtension -spdb -out "ReleaseDir\!OUTPUT_BASE_FILENAME!.msi" -loc "Lang\!PRODUCT_SKU!.Base.!CULTURE!.wxl" -loc "Lang\!PRODUCT_SKU!.!PACKAGE_TYPE!.!CULTURE!.wxl" -cultures:!CULTURE!
+
+    REM Build without extra Source Code feature
+    "!WIX!bin\heat.exe" dir "!REPRO_DIR!" -out Files-!OUTPUT_BASE_FILENAME!.wxs -gg -sfrag -scom -sreg -srd -ke -cg "AppFiles" -var var.ProductMajorVersion -var var.ProductMinorVersion -var var.ProductMaintenanceVersion -var var.ProductPatchVersion -var var.ReproDir -dr INSTALLDIR -platform !PLATFORM!
+    "!WIX!bin\candle.exe" -arch !PLATFORM! Main.wxs Files-!OUTPUT_BASE_FILENAME!.wxs -ext WixUIExtension -ext WixUtilExtension -dProductSku="!PRODUCT_SKU!" -dProductMajorVersion="!PRODUCT_MAJOR_VERSION!" -dProductMinorVersion="!PRODUCT_MINOR_VERSION!" -dProductMaintenanceVersion="!PRODUCT_MAINTENANCE_VERSION!" -dProductPatchVersion="!PRODUCT_PATCH_VERSION!" -dProductId="!PRODUCT_ID!" -dReproDir="!REPRO_DIR!" -dSetupResourcesDir="!SETUP_RESOURCES_DIR!" -dCulture="!CULTURE!"
+    "!WIX!bin\light.exe" Main.wixobj Files-!OUTPUT_BASE_FILENAME!.wixobj -ext WixUIExtension -ext WixUtilExtension -spdb -out "ReleaseDir\!OUTPUT_BASE_FILENAME!.msi" -loc "Lang\!PRODUCT_SKU!.Base.!CULTURE!.wxl" -loc "Lang\!PRODUCT_SKU!.!PACKAGE_TYPE!.!CULTURE!.wxl" -cultures:!CULTURE!
+
+
+    COPY /Y "ReleaseDir\!OUTPUT_BASE_FILENAME!.msi" "ReleaseDir\!OUTPUT_BASE_FILENAME!.MUI.msi"
+    
+    REM Generate setup translations
+    CALL BuildSetupTranslationTransform.cmd de-de 1031
+    CALL BuildSetupTranslationTransform.cmd es-es 3082
+    CALL BuildSetupTranslationTransform.cmd fr-fr 1036
+    REM CALL BuildSetupTranslationTransform.cmd it-it 1040
+    CALL BuildSetupTranslationTransform.cmd ja-jp 1041
+    REM CALL BuildSetupTranslationTransform.cmd ko-kr 1042
+    REM CALL BuildSetupTranslationTransform.cmd ru-ru 1049
+    CALL BuildSetupTranslationTransform.cmd zh-cn 2052
+    CALL BuildSetupTranslationTransform.cmd zh-tw 1028
+
+    REM Add all supported languages to MSI Package attribute
+    CSCRIPT "%ProgramFiles(x86)%\Windows Kits\%WIN_SDK_VERSION%\bin\x64\WiLangId.vbs" ReleaseDir\!OUTPUT_BASE_FILENAME!.MUI.msi Package !LANGIDS!
+
+    REM SIGN the MSIs with digital signature.
+    REM Dual-Signing with SHA-1/SHA-256 requires Win 8.1 SDK or later.
```

Comment:
I think we have the correct certs etc in this repo - are you able to attempt signing?

Reply:
I do not have access to the private keys. Signing the MSI would be great, but has not tested yet. I only added the commands for it.